googletest/src/gtest_main.cc

```diff
@@ -32,18 +32,26 @@
 #include "gtest/gtest.h"
 
 #if GTEST_OS_ESP8266 || GTEST_OS_ESP32
-#if GTEST_OS_ESP8266
```

Comment:
This change seems unrelated to the issue this CL is addressing.

Reply:
Reverted